server/liveblog/themes/themes_assets/default/js/theme/helpers.js

```diff
@@ -47,6 +47,7 @@ function getJSON(url) {
     var xhr = new XMLHttpRequest();
 
     xhr.open('GET', url);
+    xhr.setRequestHeader("Cache-Control", "no-cache");
```

Comment:
this will add no-cache to every request, isn't it? that would have a significant impact on our server side.

Reply:
I added this so that after voting, the results reflect immediately on the next cycle of checking for new posts and not after a few api calls as it is now because of the cache. 

Or do you think we can still proceed with it that way ?

Reply:
I understand the reasoning but I think this `getJSON` function is used for different things not only the polls, right?

Reply:
It's used mainly when getting all posts. 

The other usecases are: 
1. In `ads-manager` during init.
2. Getting a single post and here it is for the new share functionality.